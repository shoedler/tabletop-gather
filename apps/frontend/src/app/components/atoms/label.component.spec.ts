import { TestBed } from '@angular/core/testing';
import { LabelComponent } from './label.component';

describe(LabelComponent.name, () => {
  beforeEach(async () => {
    await TestBed.configureTestingModule({
      imports: [LabelComponent]
    }).compileComponents();
  });

  it('should create the component', () => {
    const fixture = TestBed.createComponent(LabelComponent);
    const component = fixture.componentInstance;
    expect(component).toBeTruthy();
  });

  it('should match snapshot', () => {
    const fixture = TestBed.createComponent(LabelComponent);
    fixture.componentInstance.label = 'Test Label';
    fixture.componentInstance.id = 'test-id';
    fixture.detectChanges();
    expect(fixture).toMatchSnapshot();
  });
});
