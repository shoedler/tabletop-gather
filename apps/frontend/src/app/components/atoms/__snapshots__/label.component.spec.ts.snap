// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LabelComponent should match snapshot 1`] = `
<tg-label
  id={[Function String]}
  label={[Function String]}
>
  <div
    class="tg-p-1"
  >
    <label
      class="label"
      for="test-id"
    >
      Test Label
    </label>
  </div>
</tg-label>
`;
